### PR TITLE
Changelog for 0.18.0

### PR DESCRIPTION
**Pull request by datho7561**
Signed-off-by: David Thompson <davthomp@redhat.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
...
+## [0.18.0](https://github.com/eclipse/lemminx/milestone/26?closed=1) (August 9, 2021)
+
+### Enhancements
+
+ * Added CodeLens that displays referenced grammars at the top of an XML file. See [#1092](https://github.com/eclipse/lemminx/issues/1092).
+ * Added CodeAction to bind an XML document to an existing schema. See [#1088](https://github.com/eclipse/lemminx/pull/1088).
+ * Added request 'canBindGrammar' to help implement binding an XML document to an existing schema. See [#1084](https://github.com/eclipse/lemminx/pull/1084).
+ * Let scanner check for whitespace. See [#1077](https://github.com/eclipse/lemminx/pull/1077).
+ * Provide Document Lifecycle Participant for tracking `didOpen`, `did*`. See [#603](https://github.com/eclipse/lemminx/issues/603).
+ * Report telemetry events about text documents that are opened: file extension, grammar binding strategy, grammar resolving strategy See [#1066](https://github.com/eclipse/lemminx/issues/1066).
```
Comment (datho7561) [inline CHANGELOG.md:12]: oops forgot period

**Comment by AlexXuChen** [comment]
I can review

**Review by AlexXuChen — APPROVED** [review]
LGTM

**Comment by datho7561** [comment]
Thanks for the review, Alex!

**Pull request merged**